cfde_deriva/tableschema.py

```diff
@@ -413,24 +426,32 @@ def registry_record_page(tname, rid=None):
             },
         })
         model.annotations[tag.chaise_config]['navbarMenu']['children'].append({
-            "name": "In-Review Submission",
+            "name": "Review Options",
             "acls": {
                 "enable": self.get_review_acl(),
             },
             "children": [
                 {
-                    "name": "Content Summary Charts",
+                    # header, not linkable
+                    "name": "Submission %s" % datapackage['id']
```

Comment:
Not sure if this is meant to be a header or a link, also need clarification from Amanda.

Reply:
From the mock-up in the issue, I think this is intended as the parent of the following links.  E.g. this is the particular submission and those are links to perform actions related to this submission.